Multisig/Cross-layer/Configuration/Config.Example.xcconfig

```diff
@@ -50,6 +50,11 @@ _CLAIMING_DATA_URL_DEV        = https:/$()/safe-claiming-app-data.staging.5afe.d
 _CLAIMING_DATA_URL_STAGING    = https:/$()/safe-claiming-app-data.staging.5afe.dev/
 _CLAIMING_DATA_URL_PROD       = https:/$()/safe-claiming-app-data.gnosis-safe.io/
 
+// Gelato relay service
+_RELAY_URL_DEV                = https:/$()/
```

Comment:
Can you provide a URL for testing? Or do you sue an instance running on your laptop?

Reply:
I will adjust URLs after service is deployed.